### PR TITLE
Feature/edit activities comments

### PR DESCRIPTION
**Pull request by nttuyen**

### CODE REVIEW

File: extension/config/src/main/resources/conf/platform/exo-sample.properties [hunk 1]
```diff
@@ -373,6 +373,16 @@
...
 
+#Enables Manager Edit activity:
+#exo.manager.edit.activity.enable= true
+#Enables Manager Edit comment:
+#exo.manager.edit.comment.enable= true
+
+#Enables Edit activity:
+#exo.edit.activity.enable= true
+#Enables Edit comment:
+#exo.edit.comment.enable= true
```
Comment (thomasdelhomenie) [inline extension/config/src/main/resources/conf/platform/exo-sample.properties]: Should also be defined in https://github.com/exodev/platform/blob/develop/extension/config/src/main/resources/conf/platform/configuration.properties for their default value.

File: extension/config/src/main/resources/conf/platform/configuration.properties [hunk 2]
```diff
@@ -642,6 +642,16 @@ wiki.editPage.livingTime=${exo.wiki.editPage.livingTime:1800000}
 # Spaces super managers groups
 social.spaces.administrators=${exo.social.spaces.administrators:}
 
+#Enables Manager Edit activity:
+exo.manager.edit.activity.enable= true
```
Comment (thomasdelhomenie) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: There is a space after the = which must be removed (for all properties).

File: extension/config/src/main/resources/conf/platform/exo-sample.properties (hunk 1 above)
Comment (nttuyen) [reply extension/config/src/main/resources/conf/platform/exo-sample.properties]: i'm done fixing this

File: extension/config/src/main/resources/conf/platform/configuration.properties (hunk 2 above)
Comment (nttuyen) [reply extension/config/src/main/resources/conf/platform/configuration.properties]: i'm done fixing this

**Pull request merged**